### ISSUE TITLE
See activity for which you are enrolled in schema when the activity is not (yet) visible

### ISSUE THREAD

**Issue by ThijsieNL** [svsticky/constipated-koala#1043]
### Expected behaviour
Not sure if this is a bug or intended behaviour. I'd say the expected behaviour is as follows: 
when the  board enrolls you for an activity (in my case for the dies committee), it should not be visible in your list of activities on your koala home page

### Actual behavior
you can see the activities you are enrolled for while they are not actually visible yet. 
![image](https://user-images.githubusercontent.com/93656203/230591876-d3e0c586-6b2e-4866-ad6c-a1e65eae6ce5.png)
^activities in my overview
![image](https://user-images.githubusercontent.com/93656203/230591934-6f7ea6e1-5a61-4ba4-bd89-0b7a04bb6b8e.png)
^when I click on one activity because it is not actually a visible activity yet.

### Steps to reproduce
enroll yourself in an activity that is not yet available to the public
open your home page in koala and look at your activity-page

- admin/user -> user 
- browser: chrome

**Comment by SilasPeters** [comment]
Thanks for your idea!

One could say that the behavior is correct, as the goal of the list is to notify you for which activities you are signed up - regardless of their date or visibility. I personally would like to know that I was magically signed up by somebody, so that I do not have a surprise when the activity does become visible.

What might be your reasons on why they should not be visible, other than possible confusion - which I agree upon.

Perhaps, it would be more logical if the activity itself can be seen by you when you are signed up.

TL;DR I do not have any preferences on how to address this. I believe that the current behavior makes sense.
